### DISCUSSION TITLE
Setting server keep alive timeout.

### DISCUSSION THREAD

**Discussion by XPAULUSX** [nuxt/nuxt#15962] [General]
Hi.

Our application is sat on an ELB, we're currently getting intermittent 502 errors which I believe is due to nodes default 5 second keep alive timeout?  I understand changing the ELB idle timeout to 4 seconds would resolve this but i've been advised by my colleagues this isn't possible in our instance.  The only other solution would be to change the node config.

Looking a the thread below this looks like it was possible in nuxt 2?

https://github.com/nuxt/nuxt.js/issues/9740

Is there anyway to do this in nuxt3?

Thank you

Paul

**Comment by antonyfrancis** [comment]
There is a workaround on this issue. https://github.com/nuxt/nuxt/issues/18857
